torch/testing/_internal/common_methods_invocations.py

```diff
@@ -246,6 +248,9 @@ def sample_inputs(self, device, dtype, requires_grad=False):
         """
         return self.sample_inputs_func(self, device, dtype, requires_grad)
 
+    def sample_sparse_inputs(self, device, dtype, requires_grad=False):
+        raise NotImplementedError("Please override in the appropriate derived class")
```

Comment:
Don't override it, set it as a default value for all OpInfos of the class (more on this below)

Reply:
Done as suggested as above